tests/ui/typeck/issue-115264-expr-field.rs

```diff
@@ -0,0 +1,17 @@
+// Regression test for issue 115264
```

Comment:
Can you move these two tests to `ui/attributes`? This has nothing to do with typeck.

Reply:
Done